MetadataExtractor/Formats/Exif/makernotes/OlympusMakernoteDirectory.cs

```diff
@@ -441,16 +441,8 @@ private void ProcessCameraSettings([NotNull] byte[] bytes)
             var reader = new SequentialByteArrayReader(bytes);
             var count = bytes.Length / 4;
 
-            try
-            {
-                for (var i = 0; i < count; i++)
-                    Set(CameraSettings.Offset + i, reader.GetInt32());
-            }
-            catch (IOException e)
-            {
-                // Should never happen, given that we check the length of the bytes beforehand.
-                Debug.WriteLine(e);
-            }
+            for (var i = 0; i < count; i++)
+                Set(CameraSettings.Offset + i, reader.GetInt32());
         }
 
         public bool IsIntervalMode()
```

Comment:
Agree. This is likely a remnant from the Java version where checked exceptions can't determine this kind of thing.